src/table/src/components/TableHeader.js

```diff
@@ -38,15 +38,20 @@ const TableHeader = (props :Props) => {
       <StyledRow sticky={sticky}>
         {
           headers && headers.map((header) => {
-            const { key, label, cellStyle } = header;
+            const {
+              cellStyle,
+              key,
+              label,
+              sortable = true
+            } = header;
             return (
               <components.HeadCell
                   key={key}
                   components={components}
                   cellStyle={cellStyle}
-                  onClick={onSort ? createSortHandler(key) : undefined}
+                  onClick={(onSort && sortable) ? createSortHandler(key) : undefined}
                   order={orderBy === header.key ? order : false}
-                  sortable>
+                  sortable={sortable !== false}>
```

Comment:
Why not `sortable === true`?

Reply:
oops. Left over from when `sortable` didn't have a default value.